src/main/fc/rc.c

```diff
@@ -133,6 +137,11 @@ float getRawSetpoint(int axis)
     return rawSetpoint[axis];
 }
 
+float getRcCommandDelta(int axis)
+{
```

Comment:
```suggestion
float getRcCommandDelta(int axis) {
```

Reply:
Gosh I'd have to do that through hundreds of places.
I will be revising the nomenclature of rc.c later, how about I do it then, with advice from Mike?

Reply:
No. This is the correct use of braces for functions according to https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md